### PR TITLE
Fix compass benchmark job

### PR DESCRIPTION
**Pull request by PetarTodorovv**
**Description**
With some recent changes, the benchmark job is not adapted and now it's failing. The missing `compass-postgresql` secret is not created.
Also, there was a problem with the istio and certificate configuration, the following error was logged in the `istiod` pod: `2022-09-16T07:26:47.126805Z	warn	ads	ADS:LDS: ACK ERROR istio-ingressgateway-6d9875cd44-ntlfk.istio-system-69 Internal:Error adding/updating listener(s) 0.0.0.0_8443: Failed to load certificate chain from <inline>, only RSA certificates with 2048-bit, 3072-bit or 4096-bit keys are supported in FIPS mode`

Changes proposed in this pull request:
- Install postgresql charts before compass installation
- change the generated private key to be 4096 bit instead of 4069

**Related issue(s)**
- https://github.com/kyma-incubator/compass/pull/2568

### CODE REVIEW

**Review by dzahariev — APPROVED** [review]

File: vpath/pjtester.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+pjNames:
+  - pjName: "pre-main-compass-gke-benchmark"
+    pjPath: "test-infra/prow/jobs/incubator/compass/"
+    report: true
+prConfigs:
+  kyma-incubator:
+    compass:
+      prNumber: 2596
```
Comment (dzahariev) [inline vpath/pjtester.yaml:8]: 2616

**Review by dzahariev — CHANGES_REQUESTED** [review]

**Review by ivantenevvasilev — APPROVED** [review]

**Review by DimitarPetrov — APPROVED** [review]

**Pull request merged**